exporter/elasticsearchexporter/model.go

```diff
@@ -155,6 +161,131 @@ func (m *encodeModel) encodeLogECSMode(resource pcommon.Resource, record plog.Lo
 	return document
 }
 
+func (m *encodeModel) encodeMetrics(resource pcommon.Resource, metrics pmetric.MetricSlice, _ pcommon.InstrumentationScope) ([][]byte, error) {
+	hasher := sha256.New()
```

Comment:
Is using a cryptographic hash necessary? I think we should probably use a fast, non-cryptographic hash algorithm like xxhash (specifically github.com/cespare/xxhash/v2)

Reply:
Very good point. This reminds me that [the OTel Collector contributing guide](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.102.1/CONTRIBUTING.md#recommended-libraries--defaults) specifically recommends [hash/fnv](https://pkg.go.dev/hash/fnv).

Reply:
Fixed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33513/commits/cf4d392c28e6f67ba9cbee80fa211f887ff11a11.